src/components/Attachments/AttachmentView/AttachmentViewPdf/index.native.js

```diff
@@ -15,6 +20,8 @@ function AttachmentViewPdf({file, encryptedSourceUrl, isFocused, isUsedInCarouse
             if (isUsedInCarousel) {
                 const shouldPagerScroll = scale === 1;
 
+                attachmentCarouselPagerContext.onPinchGestureChange(scale !== 1);
```

Comment:
Since we set `shouldPagerScroll` to be `scale === 1` wouldn't it be easier to do this
```suggestion
                attachmentCarouselPagerContext.onPinchGestureChange(!shouldPagerScroll);
```